core/commands/stat.go

```diff
@@ -116,14 +150,14 @@ var statBwCmd = &cmds.Command{
 					totals := nd.Reporter.GetBandwidthTotals()
 					out <- &totals
 				}
-				if !doPoll {
-					return
-				}
-				select {
```

Comment:
this change isnt right. The existing logic says to halt execution after the first print if we arent in polling mode.